odysseus/odysseus_tree/sources/tpu_telemetry/telemetry/README

```diff
@@ -0,0 +1,14 @@
+In order to run the virtual environment from Powershell, run the following commands:
+    - python3 -m venv env
+    - activate
+    - pip install -r C:\Users\matth\Siren\odysseus\odysseus_tree\sources\tpu_telemetry\requirements.txt
```

Comment:
please make all paths relative and with forward slashes, so `C:\Users\matth\Siren\odysseus\odysseus_tree\sources\tpu_telemetry\requirements.txt `becomes `./Siren/odysseus/odysseus_tree/sources/tpu_telemetry/requirements.txt`

Do this in line 4 and 7